aws/requests.yaml

```diff
@@ -1,4 +1,9 @@
 releases:
+    - name: "> 12.7.0, < 13.0.0"
```

Comment:
we alre already drafting 13.0.0 so does this make sense?

there is no planned release 12.8.0 for now

Reply:
ok, bumped it to 13.0.0